integration/tests/tracer/test_tracer_debug_methods.py

```diff
@@ -460,3 +513,17 @@ def test_debug_get_raw_transaction_non_existent_tx_hash(self):
             response["error"]["message"]
             == "Empty Neon transaction receipt for 0xd9765b77e470204ae5edb1a796ab92ecb0e20fea50aeb09275aea740af7bbc69"
         )
+
+    def test_debug_trace_call_to_precompiled_contract(self, pytestconfig, eip1052_checker):
+        sender_account = self.accounts[0]
+        tx = self.web3_client.make_raw_tx(sender_account)
+        precompiled_acc = AccountData(address="0xFf00000000000000000000000000000000000004")
+
+        instruction_tx = eip1052_checker.functions.getContractHashWithLog(precompiled_acc.address).build_transaction(tx)
```

Comment:
in this test you shoud call some function of 0xFf00000000000000000000000000000000000004
eip1052_checker is not precompiled contract and nowyou don't call precompiled contract at all